splunk-otel-android/src/main/java/com/splunk/rum/ExplicitVisibleScreenNameTracker.java

```diff
@@ -0,0 +1,39 @@
+package com.splunk.rum;
+
+import androidx.annotation.Nullable;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import io.opentelemetry.android.instrumentation.activity.VisibleScreenTracker;
+
+public class ExplicitVisibleScreenNameTracker extends VisibleScreenTracker {
```

Comment:
I think that this type of polymorphic inheritance can become problematic. Would much prefer to use delegation, and I think that long term perhaps `VisibleScreenTracker` needs to migrate to an interface (or two). I also wouldn't want to count on `VisibleScreenTracker` remaining an external API class for too long, things are in flux over there.

Reply:
Hm, how would you implement delegation here? For example when we construct `StartTypeAwareMemorySpanBuffer` it wants an instance of `VisibleScreenTracker`, same with `ScreenAttributesSpanProcessor`, right? So with the existing API I need something that `extends VisibleScreenTracker` anyway? I'm open to ideas.

Reply:
Okay! Moved the logic to a span processor, so we don't have the inheritance anymore.